src/api/services/utils.py

```diff
@@ -4,6 +4,11 @@
 
 
 def send_slack_message(webhook, body):
-  r = requests.post(url = webhook, data = json.dumps(body)) 
+  # fix for sending to Super Admin webhook.
+  # it needs to have a text field, so just stick the json string in there
+  if not body.get("text",None):
+    body["text"] = json.dumps(body)
+
+  r = requests.post(url = webhook, data = json.dumps(body))
```

Comment:
How did you create the webhook?  Is it a webhook created from Workflow Builder?  If not check this out https://slack.com/help/articles/360041352714-Create-more-advanced-workflows-using-webhooks

Reply:
I set it up according to instructions onlin through an App i created called SuperAdminNotify.
The webhook you set up for a community admin channel doesn't work.